src/Toml/Edsl.hs

```diff
@@ -0,0 +1,26 @@
+module Toml.Edsl
+       ( toml
+       , (=:)
+       , table
+       ) where
+
+import Control.Monad.State (State, execState, modify)
+
+import Toml.PrefixTree (Key)
+import Toml.Type (AnyValue (..), TOML (..), Value (..))
+
+import qualified Data.HashMap.Strict as HashMap
+import qualified Toml.PrefixTree as Prefix
+
+type Env = State TOML ()
```

Comment:
Maybe it's better to name this type somehow else? Since it EDSL for creating toml, we could name it like `TomlDsl` or `Tdsl` or `TDSL` or something like that.